tests/integration_tests/modules/test_ssh_auth_key_fingerprints.py

```diff
@@ -71,5 +71,5 @@ def test_no_home_directory_created(client: IntegrationInstance):
     assert "system" not in home_output
 
     passwd = client.execute("cat /etc/passwd")
-    assert re.match("nch:", passwd)
-    assert re.match("system:", passwd)
+    assert re.search("nch:", passwd, re.MULTILINE)
+    assert re.search("system:", passwd, re.MULTILINE)
```

Comment:
Just add a leading ^ to make sure we match lines that explicitly start with `nch:` and not `inch:` :)
```suggestion
    assert re.search("^nch:", passwd, re.MULTILINE)
    assert re.search("^system:", passwd, re.MULTILINE)
```

Reply:
good grief! I thought I had it. Sorry! adding one more commit now...